test/e2e/puppeteer-store-pagination-jquery/test.mjs

```diff
@@ -1,4 +1,6 @@
-import { getTestDir, getStats, getDatasetItems, run, expect, validateDataset } from '../tools.mjs';
+import { getTestDir, getStats, getDatasetItems, run, expect, validateDataset, skipTest } from '../tools.mjs';
+
+skipTest('Unstable test in CI, locally it works if your internet and machine is fast enough.');
```

Comment:
is it really just about the CI being slow? if so, why cant we adjust the tests so they work with slower hardware too?

Reply:
I'll experiment with it after i rewrite them to the new test site, I didn't wanna block this further, but tbh I don't get why its giving issues anyways. Locally at home it passes but with hotpot internet it doesn't always pass. In CI... Couldn't tell you